source/libs/parser/src/parInsertSql.c

```diff
@@ -977,6 +977,7 @@ static int32_t parseTagsClauseImpl(SInsertParseContext* pCxt, SVnodeModifyOpStmt
   bool     isParseBindParam = false;
   bool     isJson = false;
   STag*    pTag = NULL;
+  bool     allFixedValue = true;
```

Comment:
it seems no need to introduce this new variable as its value is always `!isParseBindParam`.